src/bin/www.ts

```diff
@@ -14,9 +14,8 @@ import logger from "../logger";
 /**
  * Get port from environment and store in Express.
  */
-const port = normalizePort(yargs.argv.PORT || yargs.argv._[0]);
+const port = yargs.parseSync().PORT || yargs.parseSync()._[0];
```

Comment:
i forgot to say we'd still need the normalizePort function to be applied here so have the value of the port passed in like before:
const port = normalizePort(yargs.parseSync().PORT || yargs.parseSync()._[0]);

Reply:
Addressed